### DISCUSSION TITLE
Installation via HACS

### DISCUSSION THREAD

**Discussion by holgi84** [weltenwort/home-assistant-rct-power-integration#267] [Q&A]
Moin, ich habe die Erweiterung mittels HACS heruntergeladen, leider finde ich nichts zur Konfiguration, wo ich die IP von Wechselrichter etc. Hinterlegen kann. Wo ist dies durchzuführen? 
Gruß Holger

**Comment by pfetzing** [comment]
Hi,
sobald du die Integration via Home Assistant hinzufügst wirst du nach der IP usw. gefragt.

Siehe Punkt 2 unter: https://github.com/weltenwort/home-assistant-rct-power-integration#via-the-hacs-integration.

Grüße
Stefan

**Reply by holgi84** [reply]
Danke für die schnelle Rückmeldung. Mir war es nicht bewusst, dass nach dem Download via HACS und Neustart vom HA die Integration noch installiert werden muss.

**Reply by weltenwort** [reply]
Ja, das ist leider etwas verwirrend. Der Begriff "Installation" ist ziemlich unscharf in Hinsicht auf HA Integrationen. Genau genommen führt HACS den "Download" durch und HA eine Art "Instanziierung", die auch mehrfach stattfinden kann für mehrere Geräte.

**Reply by HajoPeiTrammer** [reply]
Guten Abend,
es tut mir leid, dass ich so eine dumme Frage stellen muss, aber ich suche mir den ganzen Tag die Finger wund:
Wo kann ich denn die Installation starten, um dann anschließend auch die IP-Adresse einzugeben?
Ich habe die Integration heruntergeladen und ich habe HA neugestartet.
An welcher Stelle kann ich das heruntergeladenen RCT Power nun starten/installieren?
Gruß Malte

**Reply by weltenwort** [reply]
@HajoPeiTrammer Sorry für die verzögerte Antwort. Diesen Schritt musst Du dann im "Integrationen" Menü vornehmen:

![image](https://github.com/weltenwort/home-assistant-rct-power-integration/assets/973741/da5d9b18-4bb3-4fd6-bbea-61e98b6eec4b)

![image](https://github.com/weltenwort/home-assistant-rct-power-integration/assets/973741/87cf6381-f4e1-45e1-9168-6f289ab5d112)

![image](https://github.com/weltenwort/home-assistant-rct-power-integration/assets/973741/a6e473e9-2b0f-482d-bbc9-ef77ed76c3c1)

Dass man zwei Schritte braucht bei community-integrationen ist leider etwas unintuitiv.
